index.html

```diff
@@ -5,8 +5,8 @@
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <link rel="stylesheet" href="styles.css">
-  <script src="data.js"></script>
-  <script src="index.js"></script>
+  <script src="data.js" defer></script>
+  <script src="index.js" defer></script>
 
```

Comment:
Good! The script tag for the data.js file doesn't technically need a `defer` because it doesn't contain any code that will try to access elements from the DOM, but it's okay either way in this case!